icu4j/perf-tests/pom.xml

```diff
@@ -43,6 +44,64 @@
       <artifactId>commons-cli</artifactId>
       <version>${commons-cli.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.openjdk.jmh</groupId>
+      <artifactId>jmh-core</artifactId>
+      <version>${jmh.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.openjdk.jmh</groupId>
+      <artifactId>jmh-generator-annprocess</artifactId>
+      <version>${jmh.version}</version>
+      <scope>provided</scope>
+    </dependency>
   </dependencies>
 
+  <build>
```

Comment:
Can you please make this a profile?
So that this happens only when we want it, not all the time?

Also, please use [java-exec](exec:java) instead of shading + running java "by hand"

So we run the tests with `mvn exec:java ...`

Combined with a profile the end result would look something like this:
```
mvn exec:java -f tools/build/ -P jmh_benchmarks
```

See `icu4j/tools/build/pom.xml` with the `errorprone_report` profile for an example.